### PR TITLE
[MIG][11.0] auth_oauth_multi_token

### PR DESCRIPTION
**Pull request by fmdl**

### CODE REVIEW

**Comment by fmdl** [comment]
@alnslang

File: auth_oauth_multi_token/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline auth_oauth_multi_token/__init__.py]: coding not necessary

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by sla-subteno-it — APPROVED** [review]

**Comment by fmdl** [comment]
@pedrobaeza could you merge ? Thanks

cc @simahawk

**Comment by fmdl** [comment]
thanks

**Pull request merged**